inc/mod/pages.php

```diff
@@ -28,6 +28,7 @@ function mod_page($title, $template, $args, $subtitle = false) {
 		'title' => $title,
 		'subtitle' => $subtitle,
 		'boardlist' => createBoardlist($mod),
+		'pm' => create_pm_header(),
```

Comment:
I think this could be a problem as is. 
If the user is not logged, the current code will display "Trying to access array offset on value of type null" because $mod['id'] doesn't exists
https://github.com/vichan-devel/vichan/blob/3291dc27f957eb95037d4a4b6a708568636d4440/inc/mod/auth.php#L158-L163

Reply:
This could be fixed with
```php
if (!$mod) { 
return null;
}
```

Reply:
Should have been addressed with 39cc484fcf60b022d595627d02589d9c6c7fd45b